odoo/tests/common.py

```diff
@@ -2490,12 +2527,16 @@ def _is_matching(test_filter):
                 return False
             elif method and test_method and method != test_method:
                 return False
+            if parameter:
+                test._test_params.append(parameter)
             return True
 
-        if any(_is_matching(test_filter) for test_filter in self.exclude):
+        if any(_match(test_filter) for test_filter in self.exclude):
             return False
 
-        if any(_is_matching(test_filter) for test_filter in self.include):
-            return True
+        matched = False
 
-        return False
+        for test_filter in self.include:
+            matched |= _match(test_filter)  # iterate on all tags to accumulate all parameters
```

Comment:
The operator `|=` is not really adequate for booleans...

Reply:
Eh~~ it's overridden specifically for booleans (so is `^`), and is a legitimate way to perform a strictly-evaluated "or".